### ISSUE TITLE
[FR] Case Comments

### ISSUE THREAD

**Issue by milesflo** [TheHive-Project/TheHive#1193]
It would be good to have the ability to leave comments on Cases, timestamped and attributed to the user account that left them.

**Comment by mpotgieter** [comment]
Agree, this would really be helpful to us. In particular we have a use case where we would like to add comments via the API.

**Comment by illumiN8i** [comment]
This would be a great feature! Duplicate of #1041

**Comment by graememeyer** [comment]
+1, this seems like critical functionality to me.
